tests/integration/test_charm_metrics_failure.py

```diff
@@ -78,7 +97,13 @@ async def test_charm_issues_metrics_for_failed_repo_policy(
     )
 
     # Set the number of virtual machines to 0 to speedup reconciliation
-    await app.set_config({VIRTUAL_MACHINES_CONFIG_NAME: "0"})
+    await app.set_config(
+        {
+            VIRTUAL_MACHINES_CONFIG_NAME: "0",
+            "repo-policy-compliance-token": "",
+            "repo-policy-compliance-url": "",
```

Comment:
Is this here to avoid interference with other tests? Then a fixture with setup/teardown logic would be more appropriate.

Reply:
Good suggestion.
I will update the code.